docker/onePollerPerContainer/docker-compose.tmpl

```diff
@@ -1,23 +1,27 @@
-version: "3.5"
+version: "3.7"
+
 services:
 {{range .Pollers}}
   {{ .PollerName }}:
     image: {{ .Image }}
-    container_name: harvest_{{ .Version }}_{{ .PollerName }}
+    container_name: {{ .ContainerName }}
     restart: unless-stopped
-    {{if .Port }}
-    ports:
-      - "{{ .Port }}:{{ .Port }}"
-    {{end}}
-    stdin_open: true # docker run -i
-    tty: true        # docker run -t
-    command: ' --poller {{ .PollerName }} --loglevel {{ .LogLevel }} {{if .Port }} --promPort {{ .Port }} {{ end }} --config /opt/harvest.yml'
-    #entrypoint: ["sh", "-c", "sleep 2073600"]
+    {{- if .ShowPorts}}
+    {{ if .Port }}ports:
+      - {{ .Port }}:{{ .Port }}
+    {{- end}}
+    {{- end}}
+    command: '--poller {{ .PollerName }} {{if .Port }}--promPort {{ .Port }} {{ end }}
+    {{- if ne .LogLevel 2 }}--loglevel {{ .LogLevel }} {{ end}}--config /opt/harvest.yml'
     volumes:
       - harvest:/opt/harvest/
       - type: bind
-        source: {{ .ConfigFile }}
+        source: ./{{ .ConfigFile }}
```

Comment:
Why relative path? We want absolute path here.

Reply:
[Docker](https://stackoverflow.com/questions/42248198/how-to-mount-a-single-file-in-a-volume/59554543#59554543) turns this into an absolute path - verified on linux and macos